build/MakefileWorker.mk

```diff
@@ -314,7 +314,7 @@ ifeq ($(CPPUTEST_USE_GCOV), Y)
 	CPPUTEST_CFLAGS += -fprofile-arcs -ftest-coverage
 endif
 
-CPPUTEST_CXXFLAGS += $(CPPUTEST_WARNINGFLAGS) $(CPPUTEST_CXX_WARNINGFLAGS)
+CPPUTEST_CXXFLAGS += $(CPPUTEST_WARNINGFLAGS) $(CPPUTEST_CXX_WARNINGFLAGS)  -Wno-c++14-compat
```

Comment:
You are correct! I prefer not change in this way. Better to close it and create another pull request.